indico/migrations/versions/20220301_1714_57696d76f9b0_add_registration_visibility_override.py

```diff
@@ -0,0 +1,26 @@
+"""Add registration visibility override
+
+Revision ID: 57696d76f9b0
+Revises: 82fb6c6ac6db
+Create Date: 2022-03-01 17:14:04.837722
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '57696d76f9b0'
+down_revision = 'ef7a8b2e6737'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('registrations',
```

Comment:
you need an `op.alter_column()` to remove the `server_default` since it's not part of the model (the `default=...` there is python-side)